webapp/src/main/java/org/apache/atlas/web/service/LaunchDarklyConfig.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.atlas.web.service;
+
+import com.launchdarkly.sdk.LDContext;
+import com.launchdarkly.sdk.server.LDClient;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+public class LaunchDarklyConfig {
+    private String sdkKey;
+    private LDContext ldContext;
+    private LDClient client;
+
+    @Inject
+    public LaunchDarklyConfig() {
+        sdkKey              = System.getenv("USER_LAUNCH_DARKLY_SDK_KEY");
+        initClient();
+    }
+
+    private void initClient() {
+        if(StringUtils.isNotEmpty(sdkKey)) {
+            client      = new LDClient(sdkKey);
+        }
+    }
+
+    public void initContext(String context, String name, String key, String value) {
+       ldContext =  LDContext.builder(context)
```

Comment:
add try catch, launchdarky should not cause any outage

Reply:
yes